psycopg2_test.py

```diff
@@ -0,0 +1,23 @@
+import unittest
```

Comment:
`unittest` (at least the rudimentary usage here) should work ok on python2.6, but it won't on 2.4, so yeah, I'll tweak this to use `unittest2` and update requirements-dev.txt